apps/web/__tests__/test/editor/blocks.cy.ts

```diff
@@ -15,6 +15,12 @@ describe('Blocks', () => {
         editor.addBlockBottom();
     });
 
+    it('should check for move block logic', () => {
+      editor.checkFirstBlock();
+      editor.checkLastBlock();
+      editor.moveFirstBlock();
```

Comment:
```suggestion
    it('should move a block', () => {
      editor.blocksHaveDefaultOrder();
      editor.moveBlock();
      editor.blocksHaveNewOrder();
```
The sequence inside the test cases should read like a relatively natural set of steps. Imagine that if you give this snippet to a business person, they can understand what's going on and that it satisfies the requirement.

Reply:
- checkFirstBlock => Checking if the first block can't be moved up because it is the first block.
- checkLastBlock => Checking if the last block can't be moved down because it is the last block.
- moveFirstblock => Trying to move the first block to second position, after that checking if first block is not in 2nd position.

I don't see the problem with the tests here, I think the order is pretty ok.
What should be done here?

Reply:
I think that kind of illustrates the point I was trying to make. What you're describing is really three test cases packed into one. We should aim for this kind of structure:
```
describe('moving blocks on the page', () => {
  it('should move a block', () => {
    editor.blocksHaveDefaultOrder();
    editor.moveBlock();
    editor.blocksHaveNewOrder();
  })

  it('should not be possible to move the first block up', () => {...});
  it('should not be possible to move the last block down', () => {...});
})
```

Reply:
Understood, can you check latest commit?
Thank you!